io.openrct2.OpenRCT2.yaml

```diff
@@ -0,0 +1,80 @@
+app-id: io.openrct2.OpenRCT2
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: openrct2
+rename-appdata-file: openrct2.appdata.xml
+rename-desktop-file: openrct2.desktop
+rename-icon: openrct2
+copy-icon: true
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # Pulseaudio access
+  - --socket=pulseaudio
+  # Filesystem access - for game files
+  - --filesystem=home:ro
+  # Network access - for multiplayer
+  - --share=network
+  # OpenGL access
+  - --device=dri
+modules:
```

Comment:
I noticed there's few static libraries lying around. You should add a global or per-module `cleanup` array specifying files to delete. A global one should be something along this:

```
cleanup:
  - *.a
  - /share/cmake
```

Reply:
Done. Should `/lib/cmake` also be cleaned? It's only about 20K, so probably doesn't make much difference.

Reply:
Not like it actually matters, but in general yes as this is an application and not SDK extension/BaseApp.